recipes/the-silver-searcher/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "the-silver-searcher" %}
```

Comment:
`_` is permissible and some packages in `conda-forge` use this.

> The name of a package, without any reference to a particular version. Conda package names are normalized, and may contain only lowercase alpha characters, numeric digits, underscores, hyphens, or dots. In usage documentation, these will be referred to by package_name.

ref: https://conda.io/docs/building/pkg-name-conv.html